.github/actions/setup-chrome/action.yml

```diff
@@ -1,13 +1,9 @@
 name: 'Install Google Chrome'
 description: 'Install Google Chrome'
-    
+
 runs:
   using: "composite"
   steps:
       - name: Install google chrome
         shell: bash
-        run: |
-            wget -q https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_114.0.5735.133-1_amd64.deb
-            sudo apt install -y --allow-downgrades ./google-chrome-stable_114.0.5735.133-1_amd64.deb
-            sudo ln -s /usr/bin/google-chrome /usr/bin/chrome
-            chrome --version
\ No newline at end of file
+        run: './scripts/webdriver-update-newest/update-to-newest-webdriver.sh'
```

Comment:
these are different things, then wget was downloading browser and installing it, the script you are adding is installing web driver, that should be done after the the browser is installed. Not sure if chrome itself is already installed

Reply:
change this back to stable https://github.com/Alfresco/alfresco-ng2-components/pull/8770/files and update web driver in a different place

Reply:
here we don't  need to install chrome , we got chrome version in this run
 https://github.com/Alfresco/alfresco-ng2-components/actions/runs/9253234281/job/25453146522
 
 Can we rename this setup-chrome job to update-chrome-driver to update latest driver.